subscribers:
  soft_emergency_stop:
    topic:              /soft_emcy_stop
    queue_size:         10
  hard_emergency_stop:
    topic:              /hard_emcy_stop
    queue_size:         10
  actuator_commands:
    topic:              /smb_highlevel_controller/actuator_commands
    queue_size:         10
    latch:              false

publishers:
  soft_emergency_stop:
    topic:              /soft_emcy_stop
    queue_size:         1
    latch:              false
  active_state:
    topic:              active_state
    queue_size:         1
    latch:              true
  battery_voltage:
    topic:              /sensors/battery_voltage
    queue_size:         1
    latch:              true
  dysfunction:
    topic:              /dysfunction/state
    queue_size:         1
    latch:              true
  actuator_readings:
    topic:              /sensors/actuator_readings
    queue_size:         1
    latch:              false